nodejs/eks/nodegroup.ts

```diff
@@ -1422,6 +1422,24 @@ export type ManagedNodeGroupOptions = Omit<
      */
     clusterName?: pulumi.Output<string>;
 
+    /**
+     * Extra args to pass to the Kubelet.  Corresponds to the options passed in the `--kubeletExtraArgs` flag to
+     * `/etc/eks/bootstrap.sh`.  For example, '--port=10251 --address=0.0.0.0'.
```

Comment:
Can you explain how to escape values that contain spaces and special characters here?